package.json

```
@@ -78,7 +78,8 @@
     "extensions-ci": "node --max-old-space-size=8095 ./node_modules/gulp/bin/gulp.js extensions-ci",
     "extensions-ci-pr": "node --max-old-space-size=4095 ./node_modules/gulp/bin/gulp.js extensions-ci-pr",
     "perf": "node scripts/code-perf.js",
-    "update-build-ts-version": "yarn add typescript@next && tsc -p ./build/tsconfig.build.json"
+    "update-build-ts-version": "yarn add typescript@next && tsc -p ./build/tsconfig.build.json",
+    "fast-build": "node --max-old-space-size=8192 ./node_modules/gulp/bin/gulp.js vscode --disable-mangler"
```

This is awesome. It is saving me so much time when running local release builds!!

Thoughts on doing something like this so that `--disable-mangler` can be applied to any of the builds? 

```suggestion
    "gulp-fast": "node --max-old-space-size=8192 ./node_modules/gulp/bin/gulp.js --disable-mangler"
```

Example usage:
```
[yarn/npm run] gulp-fast vscode
[yarn/npm run] gulp-fast vscode-darwin-arm64
```

I was also thinking `gulp-fast` could be easier to remember if people are used to typing `gulp` already